### DISCUSSION TITLE
Creating a drumkit with multiple velocities and round robins

### DISCUSSION THREAD

**Discussion by Aleks-5** [hydrogen-music/hydrogen#1297] [Q&A]
Hello! I found that Hydrogen has an option for round robin layers. Is there a possibility to add multiple RR layers of samples with various velocities? For example, I have a RR1 folder with velocity of 70, then a RR2 with the same velocity, then a RR1 with the velocity of 100, and a RR2 with the same velocity etc. If it is possible, how can I do that?

**Comment by theGreatWhiteShark** [comment]
Hey @Aleks-5

In Hydrogen only those layers which are active for a given note velocity (their blue rectangle covers the velocity value) will be used for the round robin in the sample selection.

Take for instance the setup below:

![c14baf62-dfec-11e5-8306-393f094eaa43](https://user-images.githubusercontent.com/14164547/121786030-b1f7da00-cbbd-11eb-8fdb-0599b3d961b3.png)

For all notes with a velocity smaller than 0.5 one of layers to the left will be picked using round robin. For those with values larger than 0.5 one of the layers to the right will be picked.

Does this already answers your question?
